spec/index.html

```diff
@@ -536,6 +569,70 @@ <h3>RDF Documents and Syntaxes</h3>
       and different ordering of triples. While these aspects can have great
       effect on the convenience of working with the <a>RDF document</a>,
       they are not significant for its meaning.</p>
+
+    <p>The basis for <a>concrete RDF syntaxes</a> is the structure of the
```

Comment:
```suggestion
    <p>The basis for <a>concrete RDF syntaxes</a> (also called RDF serialization formats)
      is the structure of the
```

Reply:
Repeats earlier text.

Line 291 explained that `<a>concrete RDF syntaxes</a>` (preferred presumable as it is a link to a `<dfn>`)

As I read it, "serialization format" is general usage, and `<a>concrete RDF syntaxes</a>` is our defined terminolgy.

Reply:
If we accept https://github.com/w3c/rdf-concepts/pull/238#discussion_r2364611320 (applies to line 291), we don't need to repeat it by saying "(also called RDF serialization formats)".